packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js

```diff
@@ -97,7 +97,7 @@ describe('Timestamp utility functions', () => {
         nullFormat,
         timezone,
       );
-      const expectedOutput = '19 October 2018, 18:10 BST';
+      const expectedOutput = 'October 19, 2018, 6:10 PM BST';
```

Comment:
This looks like the default locale is USA English; I think we should set the default locale to be `en-gb`. You should be able to do that in the timestamp container's default props

Reply:
It looks like the default is set to `en`. https://github.com/bbc/psammead/blob/2818392271aeaa559f69cafd7ef38c0ea6713b3a/packages/containers/psammead-timestamp-container/src/timestampUtilities.js#L16-L20 This test doesn't test the rendered `TimestampContainer` it tests the showRelative function.

Reply:
Yep, but I think that this change highlights that the default is set to the wrong value and should probably be 'en-GB'

@jblaut it looks like you added the default to 'en' in the snippet above; do you have any opinions on this?